frontend/src/components/contactsList.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from 'react'
+import { CreateContactForm } from './contactsForm'
+import axios from 'axios'
+
+export interface Contact {
+  id: string
+  firstName: string
+  lastName?: string
+  email?: string
+  phoneNumber?: string
+  notes?: string
+  source?: string
+  ownerID: string
+  relationships?: ContactRelationship[]
+  importantDateEvent?: ImportantDateEvent[]
+}
+
+interface ContactRelationship {
+  id: string
+  relationshipType: string
+}
+
+interface ImportantDateEvent {
+  id: string
+  date: string
+  eventType: string
+}
+
+export const ContactsList = () => {
+  const [showCreateAContactForm, setShowCreateAContactForm] = useState<boolean>(false)
+  const [contacts, setContacts] = useState<Contact[]>([])
+
+  const toggleContactForm = () => {
+    setShowCreateAContactForm(!showCreateAContactForm)
+  }
+
+  const handleContactCreate = (newContact: Contact) => {
+    // setContacts((prevState) => [...prevState, newContact])
```

Comment:
i think this comment is the code you wanted right? rm the line 39 code instead?

Reply:
Done!